mitodl/ol-data-platform
# Standarlize profiles in micromasters intermediate

### What are the relevant tickets?




close https://github.com/mitodl/hq/issues/4118

### Description (What does it do?)

- Update the profile fields in `int__mitx__users` to coalesce MITx Online, MicroMasters, and edx.org
- Update the profile fields in `int__micromasters__*` and `int__mitx__courserun_enrollments_with_programs`(int__micromasters__course_enrollments use it)  to use the above model
- Update field description in these models and upstream marts

### How can this be tested?

Run the following should cover all the changes:

dbt build --select int__mitx__users --full-refresh
dbt build --select int__mitx__courserun_enrollments_with_programs
dbt build --select intermediate.micromasters
